DEV-Simple/views/ViewController.swift

```diff
@@ -67,7 +67,7 @@ class ViewController: UIViewController {
     var lightAlpha = CGFloat(0.2)
     var useDarkMode = false
     let statusBarStyleDarkContentRawValue = 3
-    let darkBackgroundColor = UIColor(red: 13/255, green: 18/255, blue: 25/255, alpha: 1)
+    let darkBackgroundColor = UIColor(red: 26/255, green: 38/255, blue: 52/255, alpha: 1)
```

Comment:
Ohhh myyyyy! Is this how colors are defined in Swift?! :D:D:D  #til and #icantunseeitnow

Reply:
Yeah I've always hated the 0-1 scale (instead of just 0-255) but it's the most straightforward way of doing this in the code using plain Swift (without more restructuring or extensions). In XCode's Interface Builder there are hex code and other pickers which are "nicer".

Something I mentioned to you a few days ago is an idea I discussed with @benhalpern regarding a way to better support all themes sent from the web. We would want something that would scale out so the app would handle all themes (and changes like this in the future too). We can definitely discuss this more in depth next week :)